registrations/tests/test_signup_group_put.py

```diff
@@ -100,6 +101,51 @@ def test_registration_admin_can_update_signup_group(
     assert db_signup1.last_modified_by_id is None
 
 
+@freeze_time("2023-03-14 03:30:00+02:00")
+@pytest.mark.django_db
+def test_created_admin_can_update_signup_group(registration, user, user_api_client):
+    signup_group = SignUpGroupFactory(registration=registration, created_by=user)
+
+    db_signup_group = SignUpGroup.objects.get(pk=signup_group.id)
+    assert db_signup_group.extra_info != new_signup_group_extra_info
+    assert db_signup_group.last_modified_by_id is None
+
+    signup_group_data = {
+        "registration": registration.id,
+        "extra_info": new_signup_group_extra_info,
+    }
+
+    assert_update_signup_group(user_api_client, signup_group.id, signup_group_data)
+
+    db_signup_group.refresh_from_db()
+    assert db_signup_group.extra_info == new_signup_group_extra_info
+    assert db_signup_group.last_modified_by_id == user.id
+
+
+@freeze_time("2023-03-14 03:30:00+02:00")
+@pytest.mark.django_db
+def test_non_created_admin_cannot_update_signup_group(
+    user_api_client, registration, user
+):
+    signup_group = SignUpGroupFactory(registration=registration)
+
+    db_signup_group = SignUpGroup.objects.get(pk=signup_group.id)
```

Comment:
Same here: the factory instance could be used instead of fetching from DB.

Reply:
Fixed in https://github.com/City-of-Helsinki/linkedevents/pull/719/commits/4e2de2a39a5ba84b4dd8b8eb1375dc453555fb3b